docs/csharp/language-reference/operators/arithmetic-operators.md

```diff
@@ -39,15 +40,7 @@ The following operators perform arithmetic operations with operands of numeric t
-
-:::code language="csharp" source="snippets/shared/ArithmeticOperators.cs" id="CheckedOperator":::
-
-The previous example declares that the `checked` operator executes in a `checked` context. For more information on `checked` and `unchecked` contexts, see the article on [checked and unchecked contexts](../keywords/checked-and-unchecked.md).
-
-If you provide an operator with the `checked` modifier, you must also provide an implementation of that operator without the `checked` modifier. When you provide both versions, the `checked` operator is called in a `checked` context; the operator without the `checked` modifier is called in an `unchecked` contest. For this reason, it is typically referred to as an *unchecked operator*. If you only provide the operator without the `checked` keyword, it is call in both a `checked` and `unchecked` context.
-
-Those operators are defined for the `int`, `uint`, `long`, and `ulong` types. When operands are of other integral types (`sbyte`, `byte`, `short`, `ushort`, or `char`), their values are converted to the `int` type, which is also the result type of an operation. When operands are of different integral or floating-point types, their values are converted to the closest containing type, if such a type exists. For more information, see the [Numeric promotions](~/_csharpstandard/standard/expressions.md#1147-numeric-promotions) section of the [C# language specification](~/_csharpstandard/standard/README.md). The `++` and `--` operators are defined for all integral and floating-point numeric types and the [char](../builtin-types/char.md) type. The `++`, `--`, and all compound arithmetic operators (such as `+=`), dont convert their arguments to `int` or have the result type as `int`.
+In the case of integral types, those operators (except the `++` and `--` operators) are defined for the `int`, `uint`, `long`, and `ulong` types. When operands are of other integral types (`sbyte`, `byte`, `short`, `ushort`, or `char`), their values are converted to the `int` type, which is also the result type of an operation. When operands are of different integral or floating-point types, their values are converted to the closest containing type, if such a type exists. For more information, see the [Numeric promotions](~/_csharpstandard/standard/expressions.md#1147-numeric-promotions) section of the [C# language specification](~/_csharpstandard/standard/README.md). The `++` and `--` operators are defined for all integral and floating-point numeric types and the [char](../builtin-types/char.md) type. The result type of a [compound assignment expression](#compound-assignment) is the type of the left-hand operand.
```

Comment:
I've edited the first and the last sentences in this paragraph (I guess compound assignment still converts the operands and then it converts the result back).